learn/yaml-ld.md

```diff
@@ -0,0 +1,300 @@
+  <br>
+  <div id="publ"  class="span12 text-justify">
+    <h1>YAML-LD</h1>
+  </div>
+</div>
+
+<div class="container">
+  <div class="row">
+    <div class="span3">
+      <h2><span class="icon-pencil"> </span>YAML</h2>
+      <p class="text-justify">
+        <a href="https://yaml.org">YAML</a>
+        (Yet Another Markup Language) is a language designed for
+        minimalistic and concise data representation. It is widely used for
+        configuration files and metadata, and is optimized to be written
+        and read by humans. Hence its lack of braces and support of comments.
+      </p>
+    </div>
+    <!-- End span -->
+
+    <div class="span6">
+      <h2 style="text-align: center;">A Simple Example</h2>
+      <pre>
+<span style="color: rgb(255, 122, 0);">"@context"</span>: <a href="https://json-ld.org/contexts/person.jsonld">https://json-ld.org/contexts/person.jsonld</a>
+<span style="color: rgb(255, 122, 0);">"@id"</span>: <a href="http://dbpedia.org/resource/John_Lennon">http://dbpedia.org/resource/John_Lennon</a>
+<span style="color: rgb(255, 122, 0);">name</span>: <span style="color: rgb(0, 148, 0);">John Lennon</span>
+<span style="color: rgb(255, 122, 0);">born</span>: <span style="color: rgb(0, 0, 175);">1940-10-09</span>
+<span style="color: rgb(255, 122, 0);">spouse</span>: <a href="http://dbpedia.org/resource/Cynthia_Lennon">http://dbpedia.org/resource/Cynthia_Lennon</a></pre>
+    </div>
+    <!-- End span -->
+
+    <div class="span3">
+      <h2>
+        <span>YAML-LD
+      </h2>
+      <p class="text-justify">
+        YAML-LD relies upon APIs and algorithms of JSON-LD, but applies them
+        to YAML instead of JSON. Since YAML is a superset of JSON, and JSON-LD
+        is a subset of JSON, this, in most aspects, is straightforward.
+      </p>
+    </div>
+    <!-- End span -->
+  </div>
+</div>
+
+<div class="hero-unit">
+  <h2 style="text-align:center;"><span class="icon-book icon-large"></span> Specification</h2>
+  <div class="container">
+    <div class="row">
+      <p class="text-justify offset2 span8">
+        The <strong>Specification</strong> is at Candidate stage right now. This is a formal document
+        that describes how YAML-LD maps to JSON-LD and what's different between the two.
+      </p>
+      <a class="btn" href="https://json-ld.github.io/yaml-ld/">
+        Open YAML-LD Candidate Spec
+      </a>
+    </div>
+  </div>
+  <!-- End span -->
+</div>
+
+<div class="container">
+  <div id="developers" class="row">
+    <h1 class="span12" style="text-align: center;"> <span class="icon-cog"></span> Developers</h1>
+  </div>
+  <div id="developers-description" class="row">
+    <p class="offset2 span8" style="text-align: center;">YAML-LD is available in
+      a number of popular programming environments. Each implementation of YAML-LD
+      listed below is fully conforming to the official YAML-LD specifications.
+    </p>
+  </div>
+
+  <div class="row">
+    <!--div class="span3">
```

Comment:
Feel free to just remove stuff rather than comment it out.

Reply:
Thought maybe there are tools that I missed and we could quickly get them back :) But generally yes I agree, I'll see into doing a cleanup in one of the follow up PRs.